fedbiomed/researcher/secagg/_secagg_context.py

```diff
@@ -627,9 +625,6 @@ def __init__(self, parties: List[str], experiment_id: str, secagg_id: Union[str,
                 f'{ErrorNumbers.FB415.value}: LOM, bad parameter `parties` : {parties} : need '
                 'at least 2 nodes for secure aggregation')
 
-        if not self._experiment_id:
```

Comment:
I don't understand why we want to remove this test ?

IMO it is needed to ensure the `experiment_id` is not empty string (`set_experiment_id` only validates the type)
+ we need to stay coherent with `SecaggServkeyContext`

Reply:
Hi @mvesin,
It seemed to me that this check was not necessary, and I removed it. I couldn't find a case where experiment id is empty string. I will update `SecaggServkeyContext.__init__` too.